_sass/_layout.scss

```diff
@@ -331,6 +325,36 @@
         margin-top: 40px;
     }
 
+    .activity-page {
+      padding: 0 0 20px 0;
+      background-color: #DAE3E7;
```

Comment:
given the fact that we want to adopt to mozilla style colors in the future, could you please add this as a variable?